coffee-backend/src/main/kotlin/chat/letscoffee/meeting/MeetingController.kt

```diff
@@ -0,0 +1,62 @@
+package chat.letscoffee.meeting
+
+import chat.letscoffee.exception.ResourceNotFoundException
+import chat.letscoffee.security.security.CurrentUser
+import chat.letscoffee.security.security.UserPrincipal
+import chat.letscoffee.user.UserService
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.*
+
+
+@RestController
+@RequestMapping("/meetings")
+class MeetingController(
+    private val meetingService: MeetingService,
+    private val userService: UserService
+) {
+
+    @PostMapping("/")
+    @PreAuthorize("hasRole('ROLE')")
+    fun create(
+        @CurrentUser userPrincipal: UserPrincipal
+    ): Meeting {
+        val user = userService.getUserById(userPrincipal.id)
+        return meetingService.createMeetingRoom(user)
+    }
+
+    @DeleteMapping("/{meetingRoomId}")
+    @PreAuthorize("hasRole('ROLE')")
+    fun delete(
+        @CurrentUser userPrincipal: UserPrincipal,
+        @PathVariable meetingRoomId: Long
+    ): ResponseEntity<String> {
+        val meeting = meetingService.getMeetingRoomById(meetingRoomId)
+        val user = userService.getUserById(userPrincipal.id)
+        if (meeting.owner != user) {
+            throw ResourceNotFoundException("Meeting", "id, owner", "$meetingRoomId, ${user.id}")
+        }
+        meeting.active = false
+        meetingService.update(meeting)
+
+        return ResponseEntity.ok("Meeting room $meetingRoomId deleted.")
+    }
+
+    @GetMapping("/{meetingRoomId}")
+    @PreAuthorize("hasRole('ROLE')")
```

Comment:
I was thinking about how we want to distribute the invitations to the meeting. Directly distributing the URL to clients on invitation leaves us unable to handle potential changes / problems with meetings and invitations in production. By just distributing the meetingId, we still have control of the logic in case anything goes wrong. 

I'm not sure that this endpoint is the best way to solve our problem though. Should probably have a listing endpoint for fetching all the meeting rooms for a user instead of this endpoint for a single one.

WIll remove this one, and make a new one when we have a plan for the flow in the app for listing invitations 👍